desktop/apps/editorial_features/routes.coffee

```diff
@@ -39,6 +40,8 @@ Articles = require '../../collections/articles.coffee'
 @venice = (req, res, next) ->
   @curation = new Curation(id: sd.EF_VENICE)
   @videoGuide = new Article(id: sd.EF_VIDEO_GUIDE)
+  user = res.locals.sd.CURRENT_USER
+  subscribedToEditorial user.email
```

Comment:
This assumes that `subscribedToEditorial` is synchronous, but it isn't since it needs to talk to ST. You'll have to wait for the ST callback before moving on by nesting the code below it or you can promisify the ST fetch and include it in your Q.all. http://bluebirdjs.com/docs/working-with-callbacks.html